Assets/Scripts/LifeManager.cs

```diff
@@ -35,6 +48,28 @@ public void Die()
 			mPhotonView.RPC("OnDeath", PhotonTargets.MasterClient);
 		}
 
+        public void InflictDamage(int damageAmount)
+        {
+            Assert.IsTrue(damageAmount >= 0);
+            mCurHealth = Mathf.Max(0, mCurHealth - damageAmount);
+
+            mPhotonView.RPC("UpdateLocalHealthBar", PhotonTargets.All, mCurHealth);
```

Comment:
In this particular case, this would require us to get the PhotonView every time a player is hit by a projectile. Is this what we want?